### PR TITLE
perf(simplecache): Allow 304 responses even with simplecache off

### PR DESCRIPTION
**Pull request by mrclay**
Should make it a bit faster when developing.

### CODE REVIEW

**Comment by ewinslow** [comment]
LGTM

Side-comment: At some point we should refactor this class return a Response object instead of using side effects...

File: engine/classes/Elgg/Application/CacheHandler.php [hunk 1]
```diff
@@ -94,18 +94,18 @@ public function handleRequest($path) {
 			if (!\_elgg_is_view_cacheable($view)) {
 				$this->send403("Requested view is not an asset");
 			} else {
-				echo $this->renderView($view, $viewtype);
+				$content = $this->renderView($view, $viewtype);
+				$etag = '"' . md5($content) . '""';
```
Comment (ewinslow) [inline engine/classes/Elgg/Application/CacheHandler.php]: Oops, is that double-quote meant to be there twice?

**Comment by ewinslow** [comment]
Did I break this during the #8381 fix, or is this really a new `perf` feature?

**Comment by mrclay** [comment]
It's new. md5() has some server overhead of course, but seems worth it to not send the bytes.

**Pull request merged**